README.md

```diff
@@ -1,100 +1,18 @@
-
-## Get Support
-
-* Open a [GitHub issue][issues] for bug reports, questions, or requests for enhancements.
-* Join the [GraalVM Slack][slack] to connect with the community and the GraalVM team.
-* Report a security vulnerability according to the [Reporting Vulnerabilities guide][reporting-vulnerabilities].
-
-## Repository Structure
-
-This source repository is the main repository for GraalVM and includes the following components:
-
-Directory | Description
------------- | -------------
-[`.github/`](.github/) | Configuration files for GitHub issues, workflows, ….
-[`compiler/`](compiler/) | [Graal compiler][reference-compiler], a modern, versatile compiler written in Java.
-[`espresso/`](espresso/) | [Espresso][java-on-truffle], a meta-circular Java bytecode interpreter for the GraalVM.
-[`java-benchmarks/`](java-benchmarks/) | Java benchmarks.
-[`regex/`](regex/) | TRegex, a regular expression engine for other GraalVM languages.
-[`sdk/`](sdk/) | [GraalVM SDK][graalvm-sdk], long-term supported APIs of GraalVM.
-[`substratevm/`](substratevm/) | Framework for ahead-of-time (AOT) compilation with [Native Image][native-image].
-[`sulong/`](sulong/) | [Sulong][reference-sulong], an engine for running LLVM bitcode on GraalVM.
-[`tools/`](tools/) | Tools for GraalVM languages implemented with the instrumentation framework.
-[`truffle/`](truffle/) | GraalVM's [language implementation framework][truffle] for creating languages and tools.
-[`vm/`](vm/) | Components for building GraalVM distributions.
-[`wasm/`](wasm/) | [GraalWasm][reference-graalwasm], an engine for running WebAssembly programs on GraalVM.
-
-## Related Repositories
-
-GraalVM provides additional languages, tools, and other components developed in related repositories. These are:
-
-Name         | Description
------------- | -------------
-[FastR] | Implementation of the R language.
-[GraalJS] | Implementation of JavaScript and Node.js.
-[GraalPython] | Implementation of the Python language.
-[GraalVM Demos][graalvm-demos] | Several example applications illustrating GraalVM capabilities.
-[Native Build Tools][native-build-tools] | Build tool plugins for GraalVM Native Image.
-[SimpleLanguage] | A simple example language built with the Truffle framework.
-[SimpleTool] | A simple example tool built with the Truffle framework. 
-[TruffleRuby] | Implementation of the Ruby language.
-[VS Code Extensions][vscode-extensions] | VS Code extensions for GraalVM.
+The modifications to upstream are scattered through the codebase, but are annotated with comments to help
+facilitate merge conflicts. There is also a Docker image and scripts contained in [`conclave`](conclave) to make 
```

Comment:
I guess that this link will be valid when we merge, right? Can't access it when seeing the preview

Reply:
Correct. @filipesoliveira will create this directory when he moved over the contents of `graal`.